objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
I did not want to continue... but for me is clear, and not very depending of cases...

For example ['Bernat', 32, 'Barcelona'] is right in a single column (like [23, 22, 24] from mesurements). Because if it was the first entry of more entries (the first row) I'll make a list (entry) into other list (data). I'll make [['Bernat', 32, 'Barcelona']] because later, maybe I'll have [['Bernat', 32, 'Barcelona'], ['Joan', 41, 'Barcelona']].
Oh, I'm getting older :(